src/database/models/collection.js

```diff
@@ -0,0 +1,17 @@
+import mongoose from 'mongoose';
+const {model, Schema} = mongoose;
```

Comment:
Please add a space before `model` and after `Schema`

Reply:
I've updated it